### ISSUE TITLE
grpperm.tst runs out of memory

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#1550]
In the master branch, when GAP is started with `bin/gap.sh -r -A tst/testinstall.g`, the following happens:
```
testing: /Users/alexk/GITREPS/gap-fork/tst/testinstall/grpperm.tst
########> Diff in /Users/alexk/GITREPS/gap-fork/tst/testinstall/grpperm.tst:66
# Input is:
Length(MaximalSubgroupClassReps(g));
# Expected output:
9
# But found:
Error, reached the pre-set memory limit
(change it with the -o command line option)
########
```
When all packages are loaded, or just default packages are loaded, everything works fine (that's why we do not observe this in Travis).

**Comment by stevelinton** [comment]
I think this is just a testing problem. The crucial package is tomlib. Without it gap has no choice but to compute the maximals of M24 "from scratch" which is a very large computation. With it, it essentially just looks up the answer.

I think we need to suppress this test if tomlib is not loaded, or replace it by a smaller one.

**Comment by alex-konovalov** [comment]
@stevelinton I've submitted a fix in #1551.
